src/stream/src/executor/hash_agg.rs

```diff
@@ -401,7 +401,7 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
         metrics
             .agg_cached_keys
             .with_label_values(&[&actor_id_str])
-            .set(agg_groups.values().map(|_| 1).sum());
```

Comment:
Not sure if there was some magic out of my knowledge here, but it's weird. Plz take a look @lmatz 🤣, correct me if my modification is wrong.